net/node/link.go

```diff
@@ -81,6 +81,7 @@ func (node *node) rx() error {
 			break
 		case io.EOF:
 			log.Error("Rx io.EOF ", err)
+			node.SetState(INACTIVITY)
 			goto disconnect
 		default:
 			log.Error("Read connection error ", err)
```

Comment:
the default  case also meaning the connection lost, it is better to trigger a event when the connection lost, cause  many other process or module may also have interesting in this event